docs/.map/map.csv

```diff
@@ -110,13 +110,13 @@ https://github.com/netdata/netdata/edit/master/src/collectors/README.md,Collecti
 https://github.com/netdata/netdata/edit/master/src/collectors/REFERENCE.md,Collectors configuration,Published,Collecting Metrics,
 https://github.com/netdata/agent-service-discovery/edit/master/README.md,Service discovery,Published,Collecting Metrics,
 https://github.com/netdata/netdata/edit/master/src/collectors/statsd.plugin/README.md,StatsD,Published,Collecting Metrics,
-https://github.com/netdata/netdata/edit/master/docs/observability-centralization-points/metrics-centralization-points/README.md,Metrics Centralization Points,Published,Collecting Metrics/Metrics Centralization Points,
-https://github.com/netdata/netdata/edit/master/docs/observability-centralization-points/metrics-centralization-points/configuration.md,Configuring Metrics Centralization Points,Published,Collecting Metrics/Metrics Centralization Points,
-https://github.com/netdata/netdata/edit/master/docs/observability-centralization-points/metrics-centralization-points/sizing-netdata-parents.md,Sizing Netdata Parents,Published,Collecting Metrics/Metrics Centralization Points,
-,Optimizing Netdata Children,Unpublished,Collecting Metrics/Metrics Centralization Points,
-https://github.com/netdata/netdata/edit/master/docs/observability-centralization-points/metrics-centralization-points/clustering-and-high-availability-of-netdata-parents.md,Clustering and High Availability of Netdata Parents,Published,Collecting Metrics/Metrics Centralization Points,
-https://github.com/netdata/netdata/edit/master/docs/observability-centralization-points/metrics-centralization-points/replication-of-past-samples.md,Replication of Past Samples,Published,Collecting Metrics/Metrics Centralization Points,
-https://github.com/netdata/netdata/edit/master/docs/observability-centralization-points/metrics-centralization-points/faq.md,FAQ on Metrics Centralization Points,Published,Collecting Metrics/Metrics Centralization Points,
+https://github.com/netdata/netdata/edit/master/docs/observability-centralization-points/metrics-centralization-points/README.md,Metrics Centralization Points,Published,Netdata Parents/Metrics Centralization Points,
```

Comment:
Question: have you tested these changes?

Reply:
Yes I have

Reply:
It was a bug I noticed and thought I would fix

Reply:
Yes